azure-iot-device/azure/iot/device/iothub/aio/async_clients.py

```diff
@@ -15,7 +15,7 @@
     AbstractIoTHubModuleClient,
 )
 from azure.iot.device.iothub.models import Message
-from azure.iot.device.iothub.transport import constant
+from azure.iot.device.iothub.pipeline import constant
```

Comment:
This one is correct, **don't** make a change here (being in the .aio module means it requires a full import)